sktime/forecasting/base/adapters/_neuralforecast.py

```diff
@@ -332,15 +358,47 @@ def _fit(
 
         maximum_forecast_horizon = fh.to_relative(self.cutoff)[-1]
         self._forecaster = self._instantiate_model(maximum_forecast_horizon)
-
         self._forecaster.fit(df=train_dataset, verbose=self.verbose_fit)
 
         return self
 
+    def _get_id_idx(self, indices):
+        if not isinstance(indices, pandas.MultiIndex):
+            id = 1
+            idx = indices.to_numpy()
+        else:
+            id_idx = np.array(indices.to_list())
+            id = id_idx[:, :-1].sum(axis=1)
```

Comment:
Can you please explain what does this do?

If you add an explanation comment in thecodebase, that'll be much appreciated as future references.

Reply:
comment is added.